nemo/core/classes/modelPT.py

```diff
@@ -1218,21 +1131,21 @@ def __unpack_nemo_file(path2file: str, out_folder: str) -> str:
         return out_folder
 
     @staticmethod
-    def __is_model_being_restored() -> bool:
+    def _is_model_being_restored() -> bool:
         global _MODEL_IS_RESTORED
         return _MODEL_IS_RESTORED
 
     @staticmethod
-    def __set_model_restore_state(is_being_restored: bool):
+    def _set_model_restore_state(is_being_restored: bool):
         global _MODEL_IS_RESTORED
         _MODEL_IS_RESTORED = is_being_restored
 
     @staticmethod
-    def set_eff_save(use_eff_save: bool):
+    def _set_eff_save(use_eff_save: bool):
```

Comment:
Why are these set/get methods for eff private ?

Reply:
I needed to call  _set_model_restore_state/_is_model_being_restored in NeMoArchive restore from "recipe", so had to made them "private" instead of "protected". 

Then I decided to treat all setters/getters in the same way ("scope").

If you think it is important I can make eff_save setters/getters "public" once again.

Reply:
I think the other two protected to private is fine (they kinda never should be used outside of internal code management anyway), but eff should be public. 

Once that's done let's merge

Reply:
done